Makefile

```diff
@@ -224,4 +226,4 @@ ci-install-hypershift-private:
 
 .PHONY: ci-test-e2e
 ci-test-e2e:
-	./hack/ci-test-e2e.sh
+	sh ./hack/ci-test-e2e.sh ${CI_TESTS_RUN}
```

Comment:
Is `${CI_TESTS_RUN}` valid `make` syntax? And why explicitly set a shell here rather than get the shell set through the shebang in the scipt?